examples/fabric-admin/device_manager/DeviceManager.cpp

```diff
@@ -237,6 +237,25 @@ void DeviceManager::ReadSupportedDeviceCategories()
     PushCommand(commandBuilder.c_str());
 }
 
+void DeviceManager::HandleReadSupportedDeviceCategories(chip::TLV::TLVReader * data)
```

Comment:
Generally speaking: If data can't be null here, why is this passing by pointer, not by reference?

Reply:
Yes, passing chip::TLV::TLVReader by reference is more intuitive.